src/components/portrait/portrait-gravatar.component.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MD5 from 'crypto-js/md5';
+
+class PortraitGravatar extends React.Component {
+  static propTypes = {
+    /**
+     * A custom class name for the component.
```

Comment:
I think that we could to keep the comments in one line

Reply:
@dawidzarzycki: I've now cleaned up the JSDoc blocks in e24f1e8 :thumbsup: